config/logger.conf

```diff
@@ -3,7 +3,7 @@
   "disable_existing_loggers": true,
   "formatters": {
     "brief": {
-      "format": "[%(asctime)s] - %(message)s",
+      "format": "[%(levelname)s][%(asctime)s] - %(message)s",
```

Comment:
What is the reason for the missing space here, and in the non-brief formatter? Everything is separated by spaces except these two things.

According to @gyorb he wished to put as many information as possible into a single line. Perhaps not separating loglevel and time with `][` is a better way to save space:

`[%(levelname)s %(asctime)s] - %(message)s`?